Assets/MixedRealityToolkit.Services/InputSystem/GazePointerStateMachine.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{ 
+    /// <summary>
+    /// Helper class for managing the visibility of the gaze pointer to match windows mixed reality and HoloLens 2
+    /// When application starts, gaze pointer is visible. Then when articulate hands / motion controllers
+    /// appear, hide the gaze cursor. Whenever user says the voice wake word, make the 
```

Comment:
>wake word [](start = 67, length = 9)

Nit: specify what the wake word is here (I know it's mentioned deeper in the code, as a reader of docs I would read this first and since this is hardcoded to "select" and not configurable in any way, it's probably better to have it mentioned as early as possible.

Reply:
Good catch, I forgot to update the docs.